alpenhorn/cli/group/sync.py

```diff
@@ -112,14 +112,14 @@ def _run_cancel(
     if first_time:
         if show_acqs or show_files:
             # Need to fetch the full request in this case
-            requests = ArchiveFileCopyRequest.select().where(
+            reqs = ArchiveFileCopyRequest.select().where(
```

Comment:
Was this unrelated to `mypy`, or did it want to change for some reason?

Reply:
It was not really a required change, but it was something that was confusing `mypy`.

I was also using the `requests` variable name when printing results, eg.  https://github.com/radiocosmology/alpenhorn/blob/99b37026f0be51ad78beb709a2c1fcfe03753f1a/alpenhorn/cli/group/sync.py#L129

Really I made the change to prevent confusing our future selves that may not notice these two independent uses of the same variable name.

Reply:
Ahh, makes sense. I agree that that makes it less confusing